packages/frontend-2/components/settings/workspaces/members/new/InvitesTable.vue

```diff
@@ -0,0 +1,190 @@
+      "
+    >
+      <template #name="{ item }">
+        <div class="flex items-center gap-2">
+          <UserAvatar
+            v-if="item.user"
+            hide-tooltip
+            :user="item.user"
+            light-style
+            class="bg-foundation"
+            no-bg
+          />
+          <span class="truncate text-body-xs text-foreground">{{ item.title }}</span>
+          <CommonBadge
+            rounded
+            color-classes="bg-highlight-3 text-foreground-2 capitalize"
+          >
+            {{ getRoleLabel(item.role) }}
+          </CommonBadge>
+        </div>
+      </template>
+      <template #invitedBy="{ item }">
+        <div class="flex items-center gap-2">
+          <UserAvatar hide-tooltip :user="item.invitedBy" />
+          <span class="truncate text-body-xs text-foreground">
+            {{ item.invitedBy.name }}
+          </span>
+        </div>
+      </template>
+      <!-- TODO: Add seat type -->
+      <template #seat>SEAT TYPE</template>
+      <template #lastRemindedOn="{ item }">
+        <span class="text-body-xs text-foreground-2">
+          {{ formattedFullDate(item.updatedAt) }}
+        </span>
+      </template>
+      <template #actions="{ item }">
+        <LayoutMenu
+          v-model:open="showActionsMenu[item.id]"
+          :items="actionsItems"
+          mount-menu-on-body
+          :menu-position="HorizontalDirection.Left"
+          @chosen="({ item: actionItem }) => onActionChosen(actionItem, item)"
+        >
+          <FormButton
+            :color="showActionsMenu[item.id] ? 'outline' : 'subtle'"
+            hide-text
+            :icon-right="showActionsMenu[item.id] ? XMarkIcon : EllipsisHorizontalIcon"
+            @click="toggleMenu(item.id)"
+          />
+        </LayoutMenu>
+      </template>
+    </LayoutTable>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { XMarkIcon, EllipsisHorizontalIcon } from '@heroicons/vue/24/outline'
+import { useQuery } from '@vue/apollo-composable'
+import { graphql } from '~/lib/common/generated/gql'
+import type { SettingsWorkspacesMembersInvitesTable_WorkspaceFragment } from '~/lib/common/generated/gql/graphql'
+import {
+  useCancelWorkspaceInvite,
+  useResendWorkspaceInvite
+} from '~/lib/settings/composables/workspaces'
+import { settingsWorkspacesInvitesSearchQuery } from '~/lib/settings/graphql/queries'
+import type { LayoutMenuItem } from '~~/lib/layout/helpers/components'
+import { HorizontalDirection } from '~~/lib/common/composables/window'
+import type { MaybeNullOrUndefined } from '@speckle/shared'
+
+graphql(`
+  fragment SettingsWorkspacesMembersNewInvitesTable_PendingWorkspaceCollaborator on PendingWorkspaceCollaborator {
+    id
+    inviteId
+    role
+    title
+    updatedAt
+    user {
+      id
+      ...LimitedUserAvatar
+    }
+    invitedBy {
+      id
+      ...LimitedUserAvatar
+    }
+  }
+`)
+
+graphql(`
+  fragment SettingsWorkspacesMembersInvitesTable_Workspace on Workspace {
```

Comment:
If we duplicate it we should also duplicate the fragment, it will make the queries more messy, but else there is a duplicate fragment that can have different content